CMakeLists.txt

```diff
@@ -2,6 +2,7 @@ CMAKE_MINIMUM_REQUIRED(VERSION 3.6.3 FATAL_ERROR)
 IF(COMMAND CMAKE_POLICY)
     CMAKE_POLICY(SET CMP0003 NEW) # change linker path search behaviour
     CMAKE_POLICY(SET CMP0048 NEW) # set project version
+    CMAKE_POLICY(SET CMP0057 NEW) # enable IN_LIST
```

Comment:
Please observe that we had to remove all linker-related Camps in https://github.com/eudaq/eudaq/pull/736 due to issues with ROOT dictionary generation of the StdEvtMonitor.

Reply:
CMP0057 is not linker related. It only activated evaluation of the IN_LIST operator.

Reply:
I need in Platform.cmake. In principal one get also formulate the check differently but it gets quite ugly.